src/nvim/bytes.h

```diff
@@ -0,0 +1,8 @@
+#ifndef NVIM_BYTES_H
+#define NVIM_BYTES_H
+
```

Comment:
Missing stdint include, do not forget that includes for things in `*.h.generated.h` should go here.